<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="polygonRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Polygon</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rectangleRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="quadroRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quadro</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="triangleRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Trianlgle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ellipseRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="circleRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveUpButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveDownButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveRightButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>RIGHT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveLeftButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RotateButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeCenterButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Change center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowParamsButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Show parametres</string>
    </property>
   </widget>
   <widget class="QLabel" name="parametresLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Parametres</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xLine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yLine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pointLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>(100;70)(150;30)(200;70)(180;120)(120;120)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Points:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResizeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coefLine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse / Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elAPlusButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>a+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elAMinusButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>a-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elBPlusButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>b+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="elBMinusButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>b-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangle / Qadro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rectAMinusButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>380</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>a-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rectBMinusButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>b-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rectAPlusButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>380</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>a+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rectBPlusButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>430</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>b+</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rombusRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rombus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="starRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Star</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
